Yosemite/YosemiteTests/Stores/OrderStoreTests.swift

```diff
@@ -563,6 +563,163 @@ final class OrderStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    func test_optimistic_update_order_customer_note_correctly() {
+        // Given
+        let orderStore = OrderStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+
+        let originalOrder = sampleOrder()
+        let updatedOrder = originalOrder.copy(customerNote: "Updated!")
+
+        orderStore.upsertStoredOrder(readOnlyOrder: originalOrder, in: viewStorage)
+
+        /// Update: Expected Customer note is actually coming from `order.json` (Customer note == "")
```

Comment:
Nit: This should be removed right? Since we're updating optimistically we basically ignore the response from the API.

Reply:
yep, much better 👍🏼